src/actions/order-actions.js

```diff
@@ -119,7 +128,7 @@ export const getOrderExtraQuestion = (orderExtraQuestionId) => (dispatch, getSta
 
     const params = {
         access_token: accessToken,
-        expand: "values"
+        expand: "values,sub_question_rules,sub_question_rules.sub_question_id"
```

Comment:
this expand is wrong , are u trying to get the sub_question ?
in that case is sub_question_rules.sub_question